grails-app/conf/application.yml

```diff
@@ -198,6 +198,7 @@ grails:
     databinding:
         dateFormats:
             - 'MM/dd/yyyy'
+        convertEmptyStringsToNull: false
```

Comment:
I think this would be worth putting in a separate commit with a proper commit message that would not be squashed.

Reply:
I don't really know enough about what's happening here so I'm definitely going to advocate for a separate spike for this one. What impact does it have on the API in general? I would assume that the correct behavior would be "user doesn't enter any address info, client sends a NULL address or does not include the address object". If we are including empty string for address properties does that mean we'd get a validation error since streetAddress is nulable:false? Again, my assumption would be that the address should be ignored if there are no properties set. 

If we do a spike, we should also consider using trimString. 
```
databinding.trimStrings = true
```

Reply:
This setting basically restores the Grails 1 behavior. The conversion of blank strings to null was introduced in Grails 2.3. So I think it can help keep the API consistent with this from Grails 1.

Getting back to the address binding:
In Grails 1
When we didn't enter an address we just send an empty object - I did the same in Grails 3
When we entered an address, we sent the data that we entered - I did the same
When we removed the address after saving it, we sent an object with empty strings as properties - I restored the same behavior in Grails 3

> If we are including empty string for address properties does that mean we'd get a validation error since streetAddress is nullable:false?
Yes, I got an error, because of that. Before we sent empty strings, so it works (Grails 1 data binding didn't convert empty strings to null)